client/web/wasm/Cargo.toml

```diff
@@ -12,14 +12,25 @@ crate-type = ["cdylib", "rlib"]
 default = ["console_error_panic_hook"]
 
 [dependencies]
-wasm-bindgen = "0.2.72"
-graphite-editor-core = { path = "../../../core/editor" }
-
-# The `console_error_panic_hook` crate provides better debugging of panics by
-# logging them with `console.error`. This is great for development, but requires
-# all the `std::fmt` and `std::panicking` infrastructure, so isn't great for
-# code size when deploying.
 console_error_panic_hook = { version = "0.1.6", optional = true }
+graphite-editor-core = { path = "../../../core/editor" }
+wasm-bindgen = "0.2.72"
 
 [dev-dependencies]
 wasm-bindgen-test = "0.3.22"
+
+[package.metadata.wasm-pack.profile.dev]
+wasm-opt = false
```

Comment:
I think we maybe should enable wasm-opt, it adds about 1-2 s of compile time, but might make an noticeable runtime difference
or is that to allow for debugging?

Reply:
I don't know, I think `wasm-opt` is disabled for dev builds by default anyway.

Reply:
Alright we can change it if we notice a significant difference between release and debug builds